esphome/dth22.yaml

```
@@ -0,0 +1,36 @@
+  package_import_url: github://scns/myESPhome/esphome/dth22.yaml
+
+packages:
+  remote_package:
+    url: https://github.com/scns/myESPhome
+    ref: main
+    files:
+      - esphome/package/dth22_update.yaml
+      - esphome/package/basis_esp8266.yaml #specific esp8266 basis settings
+      - esphome/package/dth22.yaml
+      - esphome/package/basis_settings.yaml
+    refresh: 0d
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)